### PR TITLE
adding ZeronMC

### PR DESCRIPTION
**Pull request by diphie**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]`)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18`)*

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

**Comment by Diphioc** [comment]
zeronmc

File: servers.json [hunk 1]
```diff
@@ -1718,5 +1718,15 @@
         "primaryAddress": "kazer.cc",
         "minecraftVersions": ["1.7", "1.8"],
         "primaryMinecraftVersion": "1.8"
+    }
```
Comment (itsJordan) [inline servers.json:1721]:
```suggestion
    },
    {
```

File: servers.json [hunk 2]
```diff
@@ -1718,5 +1718,15 @@
...
+	    {
```
Comment (itsJordan) [inline servers.json:1722]:
```suggestion
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please resolve the bracket formatting.

**Comment by Diphioc** [comment]
changed!

File: servers.json (hunk 1 above)
Comment (Diphioc) [reply servers.json]: changed!

**Review by itsJordan — DISMISSED** [review]

**Comment by GrowlyX** [comment]
Hi, @Diphioc, could you please fix the conflicts on your end?

**Comment by Diphioc** [comment]
Fixed now!

**Comment by GrowlyX** [comment]
image

**Step 1: From your project repository, check out a new branch and test the changes.**
```
git checkout -b ZeronMcDev-master master
git pull https://github.com/ZeronMcDev/ServerMappings.git master
```

**Step 2: Merge the changes and update on GitHub.**
```
git checkout master
git merge --no-ff ZeronMcDev-master
git push origin master
```

**Comment by Radi0o** [comment]
Fixed

**Comment by Radi0o** [comment]
[+] Re-Added the Logo

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please modify your addresses to not contain any subdomains.

**Comment by Radi0o** [comment]
Done

File: servers.json [hunk 3]
```diff
@@ -1731,5 +1731,15 @@
...
         "primaryMinecraftVersion": "1.7"
-    }
+    },
+    {
+        "id": "zeronmc",
+        "name": "ZeronMC",
+        "addresses": [
+            "zeronmc.ml"
+        ],
+        "primaryAddress": "play.zeronmc.ml",
```
Comment (itsJordan) [inline servers.json:1741]:
```suggestion
        "primaryAddress": "zeronmc.ml",
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please remove the subdomain from the primaryAddress field too.

File: servers.json (hunk 3 above)
Comment (Radi0o) [reply servers.json:1741]: oh sorry

**Comment by Radi0o** [comment]
Fixed

File: servers.json (hunk 3 above)
Comment (GrowlyX) [reply servers.json]: Sorry, we've made a mistake. Yes, you are able to keep your subdomain in the primaryAddress field.

**Review by itsJordan — APPROVED** [review]

**Pull request merged**